src/manual/vcs/github.md

```diff
@@ -143,22 +144,17 @@ version: 0
 tasks:
   - payload:
       maxRunTime: 3600
-      image: "ubuntu:latest"
+      image: "node:<version>"
       command:
         - "test"
     deadline: "{{ '2 hours' | $fromNow }}" # the task will timeout if it doesn't complete within 2 hours
 ```
 
-### Ubuntu-ci image
+### Node.js image
 
-A basic docker image with git, node/npm, and python/pip installed. It also
-includes a bash function `checkout-pull-request` which will automatically
-checkout and cd into a repo for testing some pull request which triggered a
-job. The container is defined at:
-https://github.com/taskcluster/taskcluster-github/tree/master/docker/ubuntu-ci
-and may be pulled from `quay.io/mrrrgn/ubuntu-ci:latest`.
+The official Node.js docker image. More information can be found [here:](https://github.com/nodejs/docker-node).
 
-The ubuntu-ci image is provided as a convenience, but *please feel free to push
+This image is provided as a convenience, but *please feel free to push
 and use your own docker images.*
```

Comment:
"This documentation uses the `node` image because it is convenient, but you may use any docker image you like, including one of your own design."